jboss-eap-cd-openshift-launch/added/launch/messaging.sh

```diff
@@ -115,140 +133,169 @@ function configure_mq() {
 # will configure the thread pool args using cgroups and the formulae provied by https://github.com/apache/activemq-artemis/blob/master/artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/client/ActiveMQClient.java
-function configure_thread_pool() {
-  source /opt/run-java/container-limits
+configure_thread_pool() {
+  if [ -a "${CONTAINER_LIMITS_INCLUDE}" ]; then
+    source ${CONTAINER_LIMITS_INCLUDE}
+  else
+    source /opt/run-java/container-limits
+  fi
   if [ -n "$CORE_LIMIT" ]; then
     local mtp=$(expr 8 \* $CORE_LIMIT) # max thread pool size
     local ctp=5                                  # core thread pool size
     JBOSS_MESSAGING_ARGS="${JBOSS_MESSAGING_ARGS}
     -Dactivemq.artemis.client.global.thread.pool.max.size=$mtp
     -Dactivemq.artemis.client.global.scheduled.thread.pool.core.size=$ctp"
+    echo "${JBOSS_MESSAGING_ARGS}"
   fi
 }
 
-# $1 - name - messaging-remote-throughput
-# <!-- ##AMQ_REMOTE_CONNECTOR## -->
-function generate_remote_artemis_remote_connector() {
-    echo "<remote-connector name=\"netty-remote-throughput\" socket-binding=\"${1}\"/>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
-}
-
-# Arguments:
-# $1 - remote context name - default remoteContext
-# $2 - remote host
-# $3 - remote port - 61616
 # <!-- ##AMQ_REMOTE_CONTEXT## -->
-function generate_remote_artemis_naming() {
-    echo "<bindings><external-context name=\"java:global/${1}\" module=\"org.apache.activemq.artemis\" class=\"javax.naming.InitialContext\">
+# deprecated, not used by the new external connector
+generate_remote_artemis_naming() {
+    declare remote_context_name="${1}"
+    declare remote_host="${2}"
+    declare remote_port="${3}"
+    echo "<bindings><external-context name=\"java:global/${remote_context_name}\" module=\"org.apache.activemq.artemis\" class=\"javax.naming.InitialContext\">
               <environment>
                   <property name=\"java.naming.factory.initial\" value=\"org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory\"/>
-                      <property name=\"java.naming.provider.url\" value=\"tcp://${2}:${3}\"/>
+                      <property name=\"java.naming.provider.url\" value=\"tcp://${remote_host}:${remote_port}\"/>
                       <!-- ##AMQ7_CONFIG_PROPERTIES## -->
               </environment>
           </external-context>
           <!-- ##AMQ_LOOKUP_OBJECTS## --></bindings>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
 }
 
-# $1 - factory name - activemq-ra-remote
-# $2 - username
-# $3 - password
-# $4 - default connection factory - java:jboss/DefaultJMSConnectionFactory
+# $1 - name - messaging-remote-throughput
+# <!-- ##AMQ_REMOTE_CONNECTOR## -->
+generate_remote_artemis_remote_connector() {
+    declare name="${1}"
+    declare socket_binding_name="${2}"
+    echo "<remote-connector name=\"${name}\" socket-binding=\"${socket_binding_name}\"/>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
+}
```

Comment:
Nit: if this would be moved back up to L129 the diff would be simpler.

Reply:
old L129, new L151 :)

Reply:
Yeah, as you pointed out below L168 should just be gone.